commerce_price_rule.info.yml

```diff
@@ -2,7 +2,7 @@ name: Commerce Price Rule
 type: module
 description: 'Provides functionality for personalising product prices based on chosen conditions.'
 package: Commerce (contrib)
-core: 8.x
+core_version_requirement: ^8.7.7 || ^9
 dependencies:
   - commerce:commerce_product
-  - system (>=8.3.0)
+  - commerce:commerce_order
```

Comment:
**CHANGE REQUEST**

@barinder Were you getting a particular error? As far as I see the module does not depend on orders.

Reply:
@krystalcode Yes, if commerce_order is not enabled following fatal error is thrown while adding price rule.
`"Drupal\Component\Plugin\Exception\PluginNotFoundException: The "commerce_order_item" entity type does not exist. in Drupal\Core\Entity\EntityTypeManager->getDefinition() (line 150 of /Users/barinder/Sites/d9/core/lib/Drupal/Core/Entity/EntityTypeManager.php)."`

Reply:
@barinder Ok I just got some time to look into this. This is [a bug](https://www.drupal.org/project/commerce/issues/3184470#comment-13914213) in the Commerce module and it should be fixed there. We should not be demanding the order module here.